packages/calypso-e2e/src/lib/blocks/block-flows/map.ts

```diff
@@ -31,17 +31,23 @@ export class MapFlow implements BlockFlow {
 	 */
 	async configure( context: EditorContext ): Promise< void > {
 		const editorCanvas = await context.editorPage.getEditorCanvas();
-		const block = editorCanvas.getByRole( 'document', { name: 'Block: Map' } );
 
 		// Enter the supplied address.
 		const editorParent = await context.editorPage.getEditorParent();
 		await editorParent.getByPlaceholder( 'Add a marker' ).fill( this.configurationData.address );
 
-		// Wait for the popover.
+		const mapkitBlock = await editorParent.locator( '[data-map-provider=mapkit]' );
```

Comment:
If I'm not mistaken, this will throw an error and terminate the test if the locator is not found.